spec/models/event_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe Event do
+  context 'after creating an event' do
+    event = Event.new
+
+    it 'should be invalid without parameters' do
+      expect(event).to_not be_valid
+    end
```

Comment:
- [ ] The test case that checks for a valid event is missing as it was requested by  @mosesogwo